benchmarks/src/btreemap.rs

```diff
@@ -219,6 +221,53 @@ pub fn btreemap_insert_10mib_values() -> BenchResult {
     })
 }
 
+#[bench]
+pub fn btreemap_iter_count() -> BenchResult {
+    let mut btree = BTreeMap::new(DefaultMemoryImpl::default());
+    let size: u8 = 200;
+    for i in 0..size {
+        btree.insert(i, i);
+    }
+
+    benchmark(|| {
+        for i in 0..size {
+            for j in i + 1..size {
+                btree
+                    .range((Bound::Included(i), Bound::Included(j)))
+                    .count();
+            }
+        }
+    })
+}
+
+#[bench]
+pub fn btreemap_iter_count_10mib_values() -> BenchResult {
+    let mut btree = BTreeMap::new(DefaultMemoryImpl::default());
+
+    // Insert 200 10MiB values.
+    let mut rng = Rng::from_seed(0);
+    let mut values = vec![];
+    for _ in 0..200 {
+        values.push(rng.iter(Rand::rand_u8).take(10 * 1024).collect::<Vec<_>>());
+    }
```

Comment:
You can also just push to the btree directly in the same loop:

```suggestion
    for i in 0..200 {
        btree.insert(i, rng.iter(Rand::rand_u8).take(10 * 1024).collect::<Vec<_>>()));
    }
```